fixtures/eurolite/tmh-w400.json

```diff
@@ -0,0 +1,159 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "TMH-W400",
+  "shortName": "tmh400",
+  "categories": ["Moving Head"],
+  "meta": {
+    "authors": ["McMars"],
+    "createDate": "2023-10-13",
+    "lastModifyDate": "2023-10-13"
+  },
+  "links": {
+    "manual": [
+      "https://www.steinigke.de/de/mpn51785930-eurolite-led-tmh-w400-moving-head-wash-zoom.html"
+    ],
+    "productPage": [
+      "https://www.steinigke.de/de/mpn51785930-eurolite-led-tmh-w400-moving-head-wash-zoom.html"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=6aR_Uofh7ms&ab_channel=eurolitevideo"
+    ]
+  },
+  "physical": {
+    "dimensions": [300, 230, 450],
+    "weight": 10,
+    "power": 400,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED",
+      "colorTemperature": 4000
+    },
+    "lens": {
+      "degreesMinMax": [9, 45]
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "capability": {
+        "type": "Pan",
+        "angle": "540deg"
+      }
+    },
+    "Pan 2": {
+      "name": "Pan",
+      "fineChannelAliases": ["Pan 2 fine"],
+      "capability": {
+        "type": "Pan",
+        "angle": "20deg",
+        "comment": "fine"
+      }
+    },
+    "Tilt": {
+      "capability": {
+        "type": "Tilt",
+        "angle": "230deg"
+      }
+    },
+    "Tilt 2": {
+      "name": "Tilt",
+      "fineChannelAliases": ["Tilt 2 fine"],
+      "capability": {
+        "type": "Tilt",
+        "angle": "20deg"
+      }
+    },
+    "Pan 3": {
+      "name": "Pan",
+      "fineChannelAliases": ["Pan 3 fine"],
+      "capability": {
+        "type": "Pan",
+        "angle": "20deg"
+      }
+    },
+    "Tilt 3": {
+      "name": "Tilt",
+      "fineChannelAliases": ["Tilt 3 fine"],
+      "capability": {
+        "type": "Tilt",
+        "angle": "20deg"
+      }
+    },
+    "Pan/Tilt Speed": {
+      "capability": {
+        "type": "PanTiltSpeed",
+        "speed": "fast"
+      }
+    },
+    "Zoom": {
+      "capability": {
+        "type": "Zoom",
+        "angleStart": "9deg",
+        "angleEnd": "48deg"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Strobe": {
+      "capability": {
+        "type": "StrobeSpeed",
+        "speedStart": "1Hz",
+        "speedEnd": "25Hz"
```

Comment:
According to the manual, it's a bit more complex than this: 0-3, 4-127, 128-191, etc, ranges each have separate strobe/pulse features.